src/pages/workspace/companyCards/WorkspaceCompanyCardsListHeaderButtons.tsx

```diff
@@ -41,6 +45,36 @@ function WorkspaceCompanyCardsListHeaderButtons({policyID, selectedFeed}: Worksp
     const isCustomFeed = CardUtils.isCustomFeed(selectedFeed);
     const companyFeeds = CardUtils.getCompanyFeeds(cardFeeds);
     const currentFeedData = companyFeeds?.[selectedFeed];
+    const policy = PolicyUtils.getPolicy(policyID);
+
+    const [list] = useOnyx(`${ONYXKEYS.COLLECTION.WORKSPACE_CARDS_LIST}${workspaceAccountID}_${selectedFeed}`);
+    const filteredCardList = CardUtils.getFilteredCardList(list);
+
+    const handleAssignCard = () => {
+        const data: Partial<AssignCardData> = {
+            bankName: selectedFeed,
+        };
+
+        let currentStep: AssignCardStep = CONST.COMPANY_CARD.STEP.ASSIGNEE;
+
+        if (Object.keys(policy?.employeeList ?? {}).length === 1) {
+            const userEmail = Object.keys(policy?.employeeList ?? {}).at(0) ?? '';
+            data.email = userEmail;
+            const personalDetails = PersonalDetailsUtils.getPersonalDetailByEmail(userEmail);
```

Comment:
NAB Let's create a util function to get card member name

Reply:
It'll be done in my other PR https://github.com/Expensify/App/pull/52735 - if it's a NAB here I will improve it in the other PR after this one is merged